encoding/thrift/inbound.go

```diff
@@ -209,3 +202,28 @@ func closeReader(r io.Reader) error {
 
 	return closer.Close()
 }
+
+// getBody avoids redundant copy if the transport request is already
```

Comment:
`getBody avoids redundant copy if the transport request is already` -> this is not what the method does, but why you have created this method. You should only explain in the comment what the method does